.github/workflows/linux.yml

```diff
@@ -22,3 +22,72 @@ jobs:
         with:
           name: ooniprobe-linux-amd64
           path: ./CLI/linux/amd64/ooniprobe
+
+  armhf_build:
+    runs-on: "ubuntu-latest"
+    name: Build on armhf Buster
+    steps:
+      - uses: actions/checkout@v2.1.0
+      - uses: uraimo/run-on-arch-action@v2.0.5
+        name: Build ooniprobe
+        id: runcmd
+        with:
+          arch: armv7
+          distro: buster
```

Comment:
Why not building on Alpine? The `alpine_latest` target supports all the arm platforms we care about. By doing that, we produce a statically linked executable that has no `libc` issues whatsoever. (Alternatively, see my suggestion below, but I would be happier with running this code inside Alpine.)